### PR TITLE
Feature/styles and routing

### PR DESCRIPTION
**Pull request by Ksenia-Tulikova**
Add styles to App Bar(Header) and Footer.
Styles was added  using JSS and MaterialUI.
Add routing to Header App Bar(Header) and Footer

### CODE REVIEW

File: client/src/app_data/pages_info/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Home from '../../components/Home';
+import WorldWeather from '../../components/WorldWeather';
+import Feedback from '../../components/Feedback/';
+import AboutUs from '../../components/AboutUs/';
+
+export const app_name = 'Weather App';
```
Comment (anton-vlasik) [inline client/src/app_data/pages_info/index.js]: By lots of conventions it is better to name constants in upper case. 
APP_NAME, PAGES_INFO.... ROUTES maybe?

File: client/src/components/AppBar/style.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import { styled } from '@mui/material/styles';
+import BaseTypography from '@mui/material/Typography';
+import BaseBox from '@mui/material/Box';
+import IconBtn from '@mui/material/IconButton';
+import AppBar from '@mui/material/AppBar';
+import MenuUI from '@mui/material/Menu';
+import ButtonMUI from '@mui/material/Button';
+import { NavLink } from 'react-router-dom';
+
+const WHITE_COLOR_GENERAL = '#fff';
```
Comment (anton-vlasik) [inline client/src/components/AppBar/style.js]: I suggest to keep colors separately somewhere in a dedicated module. Also use unified approach for color values. I suggest to change '#fff' to '#ffffff'

File: client/package.json [hunk 3]
```diff
@@ -18,6 +18,10 @@
     "npm": ">=3"
   },
   "dependencies": {
+    "@emotion/react": "^11.7.0",
```
Comment (anton-vlasik) [inline client/package.json]: do you need @emotion?

Comment (Ksenia-Tulikova) [reply client/package.json]: I've deleted it

File: client/src/components/AppBar/style.js (hunk 2 above)
Comment (Ksenia-Tulikova) [reply client/src/components/AppBar/style.js]: Fixed

File: client/src/app_data/pages_info/index.js (hunk 1 above)
Comment (Ksenia-Tulikova) [reply client/src/app_data/pages_info/index.js]: Fixed

**Pull request merged**